cmd/weirproxy/main.go

```diff
@@ -57,6 +57,7 @@ func main() {
 
 		zapcfg := zap.NewDevelopmentConfig()
 		zapcfg.Encoding = cfg.Log.Encoder
+		zapcfg.DisableStacktrace = true
```

Comment:
While `Encoder` can ensure there is no multiline string, the stacktrace from zap iteself may escape.